scripts/common.py

```diff
@@ -121,6 +121,9 @@
 REDIS_CLI_PATH = os.path.join(
     OPPIA_TOOLS_DIR, 'redis-cli-%s' % REDIS_CLI_VERSION,
     'src', 'redis-cli')
+# The directory used to store/retrieve the data/config for the emulator.
+CLOUD_DATASTORE_EMULATOR_DATA_DIR = (
+    os.path.join(CURR_DIR, 'cloud_datastore_emulator_cache'))
```

Comment:
This puts it in oppia/, right? I think @DubeySandeep was talking about putting it in a directory that's sibling to Oppia, similar to karma_coverage_reports.

Reply:
Done.